README.md

```diff
@@ -88,6 +88,22 @@ which usually is the source for security checks.
 
 Instead, the projects (i.e., for TYPO3 installations) need to have the security checks.
 
+## Documentation rendering
+
+In order to render documentation, first of all, clone repository
+
+```
```

Comment:
Please let's add the `bash` syntax highlighting to the new code blocks in this file.